docs/config_search.md

```diff
@@ -174,3 +179,21 @@ the maximal objective value within the specified constraints. In the majority of
 this will find greater than 95% of the maximum objective value (that could be found using a brute force search), while needing to search less than 10% of the configuration space.
 
 After it has found the best config(s), it will then sweep the top-N configurations found (specified by `--num-configs-per-model`) over the default concurrency range before generation of the summary reports.
+
+## Multi-Model Search Mode
```

Comment:
Can we find somewhere to put in an example run command and/or quick start?  Copy an add_sub to add_sub2 and run? Or create a new simple example model.

Reply:
Also, what are your thoughts on creating a new multi-model page instead of adding to config_search.md? Do you think it is similar enough to keep it there? Or should we give the new feature its own page (and a clean place to put examples)?